ossec-hids-local/scripts/ossec-hids-local.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=OSSEC HIDS (Observable Networks distribution)
+After=network.target 
+
+[Service]
+Type=oneshot
```

Comment:
OSSEC has [systemd unit files](https://github.com/ossec/ossec-hids/pull/619), but they're [not fully integrated yet](https://github.com/ossec/ossec-hids/issues/741). This `oneshot` service behaves a little better than the Upstart script we already have, however.